src/components/PoolForm/PoolForm.tsx

```diff
@@ -163,25 +170,22 @@ const PoolForm: FC<Props> = ({
         <PositionItem>
           <div>Position Size</div>
           <div data-cy="pool-position">
-            {formatUnits(totalPosition.add(stakedPosition), decimals).replace(
-              "-",
-              ""
-            )}{" "}
+            {formatEther(
+              convertToLP(totalPosition.add(stakedPosition))
+            ).replace("-", "")}{" "}
```

Comment:
I think this change and the fees earned change are the only displayed values that need to be touched in this PR.
```suggestion
            {formatUnits(totalPosition.add(convertFromLP(stakedPosition)), decimals).replace(
              "-",
              ""
            )}{" "}
```

Reply:
Thoughts on this suggestion?